electron/child.html

```diff
@@ -73,19 +72,21 @@
 		<div id="childTab-medical">
 			<form>
 				<h4>Basic Medical Information</h4>
-				Diagnosis: <select id="medical_condition" name="Diagnosis" class="chosen-select" multiple></select><br>
-				Significant Medical History: <textarea id="medical_history" rows="3" cols="70"></textarea><br>
+				Diagnosis: <select id="medical_condition" class="chosen-select" multiple></select>
+				<button id="submit_medical_conditions" type="button" onclick="updateMedicalConditions()">Update medical conditions</button><br>
+				Significant Medical History: <textarea id="medical_history" rows="3" cols="70"></textarea>
+				<button id="submit_medical_history" type="button" onclick="updateMedicalHistory()">Update medical history</button><br>
 				<h4>Medication</h4>
 				<table id="child_medication_table">
 					<tr><th>Medication Name</th><th colspan="3">Dosage</th><th>Start Date</th><th>End Date</th><th>Stopped</th></tr>
 					<tr>
 						<td><select data-placeholder="Add condition..." id="medication_new" class="chosen-select"></select></td>
-						<td><input type="number" id="dosage1_new" step="0.5"></td>
-						<td><input type="number" id="dosage2_new" step="0.5"></td>
-						<td><input type="number" id="dosage3_new" step="0.5"></td>
+						<td><input type="number" id="dosage1_new" step="0.5" min=0></td>
+						<td><input type="number" id="dosage2_new" step="0.5" min=0></td>
+						<td><input type="number" id="dosage3_new" step="0.5" min=0></td>
```

Comment:
User can type in any float, this only affects what the up/down arrow buttons in the number select do